packages/jest-cli/src/reporters/notify_reporter.js

```diff
@@ -37,10 +40,21 @@ export default class NotifyReporter extends BaseReporter {
     this._context = context;
   }
 
-  onRunComplete(contexts: Set<Context>, result: AggregatedResult): void {
+  async onRunComplete(
+    contexts: Set<Context>,
+    result: AggregatedResult,
+  ): Promise<any> {
     const success =
       result.numFailedTests === 0 && result.numRuntimeErrorTestSuites === 0;
 
+    const packagePath = path.resolve(
+      this._globalConfig.rootDir,
+      'package.json',
+    );
+    const {name: packageName = this._globalConfig.rootDir} = JSON.parse(
+      await readFile(packagePath),
```

Comment:
This file should be available in hastefs already, should not be necessary to hit the file system

Reply:
I'll check it out.